mora/service/common.py

```diff
@@ -17,15 +17,25 @@
 import enum
 import datetime
 import functools
-
+import uuid
 from typing import Callable, List, Tuple
 
 import flask
 import iso8601
 
 from mora import util
-from .. import lora
 from . import keys
+from .. import lora
+
+RELATION_TRANSLATIONS = {
+    'engagement': keys.ENGAGEMENT_KEY.lower(),
+    'association': keys.ASSOCIATION_KEY.lower(),
+    'it': keys.ITSYSTEM_KEY.lower(),
+    'role': keys.ROLE_KEY.lower(),
+    'address': keys.ADDRESS_KEY.lower(),
+    'manager': keys.MANAGER_KEY.lower(),
+    'leave': keys.LEAVE_KEY.lower(),
+}
```

Comment:
This feels a bit too specific — don't we have an existing map that almost does the same?

Reply:
We have a map containing all of the keys for the org funk objects, yes. Extending that with 'address' and 'it' breaks stuff, though.